app/authentication/auth.py

```diff
@@ -123,12 +128,17 @@ def _decode_jwt_token(auth_token, api_keys, service_id=None):
             decode_jwt_token(auth_token, api_key.secret)
         except TokenExpiredError as e:
             err_msg = "Error: Your system clock must be accurate to within 30 seconds"
+            extra = {
+                "error_message": err_msg,
+                "token_iat": e.token.get("iat"),
+                "local_unix_time": int(time.time()),
+                "cloudfront_request_id": request.headers.get("x-amz-cf-id"),
+            }
             current_app.logger.info(
-                'Rejecting user authentication with "%s" (token.iat: %s, us: %s) [X-Amz-Cf-Id: %s]',
-                err_msg,
-                e.token.get("iat"),
-                int(time.time()),
-                request.headers.get("x-amz-cf-id"),
+                "Rejecting user authentication with %(error_message)r "
```

Comment:
What's the purpose of the `%r` here, given `error_message` is a string? Is it just to add quotes?